logever/src/main/java/ai/smartfac/logever/controller/FormDataController.java

```diff
@@ -0,0 +1,96 @@
+package ai.smartfac.logever.controller;
+
+import ai.smartfac.logever.entity.*;
+import ai.smartfac.logever.model.LogEntry;
+import ai.smartfac.logever.service.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+@RestController
+@RequestMapping("/entry")
+public class FormDataController {
+
+    @Autowired
+    FormService formService;
+
+    @Autowired
+    UserService userService;
+
+    @Autowired
+    DepartmentService departmentService;
+
+    @Autowired
+    RoleService roleService;
+
+    @Autowired
+    FormDataService formDataService;
+
+    @PostMapping("/{form}")
+    public ResponseEntity<?> logEntry(@PathVariable(name = "form") String form,@RequestBody LogEntry logEntry) {
```

Comment:
Can we use id instead of name? It becomes difficult to apply filter based on name.